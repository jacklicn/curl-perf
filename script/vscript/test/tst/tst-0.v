var a = 15 , b = "???"

print ( "God is great" )

print ( "this var is " .. a .. "\n")
print ( "this var is " .. b .. "\n")

