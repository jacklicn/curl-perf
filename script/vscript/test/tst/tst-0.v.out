*** syntax ok ***
*** semantics and code generation ok ***

>God is great

>this var is 15


>this var is ???

